ginga/rv/plugins/Catalogs.py

```diff
@@ -646,19 +646,19 @@ def filter_results(self, starlist, filter_obj):
         if filter_obj:
             num_cat = len(starlist)
             self.logger.debug("number of incoming stars=%d" % (num_cat))
-            # TODO: vectorize wcs lookup
-            coords = [image.radectopix(star['ra_deg'], star['dec_deg'])
-                      for star in starlist]
-            arr = numpy.array(coords)
-            self.logger.debug("arr.shape = %s" % str(arr.shape))
+            coords = numpy.asarray([(star['ra_deg'], star['dec_deg'])
```

Comment:
> are there any changes that are not backward compatible?

In terms of user-facing functions, only a few of the canvas API are affected, and these are preserved by compatibility functions at the bottom of `CanvasObject`.  If someone was hacking in their own transforms into the chain then yes, that would be affected.  But that is all fairly new code to ginga and still all fairly "internal".